Code/Engine/Foundation/Types/ArrayPtr.h

```diff
@@ -213,11 +214,26 @@ class ezArrayPtr
     return ezMemoryUtils::IsEqual(static_cast<const ValueType*>(GetPtr()), static_cast<const ValueType*>(other.GetPtr()), GetCount());
   }
 
-  /// \brief Compares the two arrays for inequality.
-  EZ_ALWAYS_INLINE bool operator!=(const ezArrayPtr<const T>& other) const // [tested]
+#if EZ_DISABLED(EZ_USE_CPP20_OPERATORS)
+  template <typename = typename std::enable_if<std::is_const<T>::value == false>>
+  inline bool operator!=(const ezArrayPtr<const T>& other) const // [tested]
   {
     return !(*this == other);
   }
+#endif
+
+  /// \brief Compares the two arrays for equality.
+  inline bool operator==(const ezArrayPtr<T>& other) const // [tested]
```

Comment:
Need an overload for comparing with non-const T. Not sure why we didn't need this before.